<launch>
    <node name="lsm10_v2" pkg="lsm10_v2" type="lsm10_v2" output="screen">
        <param name="scan_topic" value="scan"/>         #设置激光数据topic名称
        <param name="frame_id" value="laser_link"/>     #激光坐标
        <param name="serial_port" value="/dev/ttyUSB0"/>  #雷达连接的串口
        <param name="baud_rate" value="460800" />        #雷达连接的串口波特率
	<param name="angle_disable_min" value="0.0"/>
	<param name="angle_disable_max" value="0.0"/>
	<param name="min_range" value="0.0"/>
	<param name="max_range" value="100.0"/>
    </node>
</launch>

